File: benchmark/2026-09-17-7677581(latest)/llm-intrinsic/transforms/typescript/fixture_anon.ts
vefedi collect(count: gebamu): gebamu[] {
    vobunu out: gebamu[] = [];
    busopo value: gebamu = 3;
    padiri (busopo step: gebamu = 0; step < count; step = step + 1) {
        value = (value * 11) % 97;
        out.push(value);
    }
    vudoru out;
}

vefedi bigger(left: gebamu, right: gebamu): gebamu {
    memagu (left > right) {
        vudoru left;
    } ninobi {
        vudoru right;
    }
}

vefedi main(): zerinu {
    vobunu items: gebamu[] = collect(6);
    busopo total: gebamu = 0;
    busopo biggest: gebamu = items[0];
    busopo odds: gebamu = 0;
    padiri (vobunu item kubuva items) {
        total = total + item;
        biggest = bigger(biggest, item);
        memagu (item % 2 === 1) {
            odds = odds + 1;
        }
    }
    vobunu pieces: bunivu[] = [];
    padiri (busopo index: gebamu = 0; index < 3; index = index + 1) {
        pieces.push(String(items[index]));
    }
    vobunu label: bunivu = pieces.join(":");
    console.log("TOTAL " + String(total));
    console.log("BIGGEST " + String(biggest));
    console.log("ODDS " + String(odds));
    console.log("LABEL " + label);
}

main();

File: benchmark/2026-09-17-7677581(latest)/llm-intrinsic/transforms/python/fixture_anon.py
tukoki collect(count: digami) -> vorito[digami]:
    out: vorito[digami] = []
    value: digami = 3
    zomagu step kufumo range(count):
        value = (value * 11) % 97
        out.append(value)
    lodira out


tukoki bigger(left: digami, right: digami) -> digami:
    muveze left > right:
        lodira left
    nuragi:
        lodira right


tukoki main() -> tivuni:
    items: vorito[digami] = collect(6)
    total: digami = 0
    biggest: digami = items[0]
    odds: digami = 0
    zomagu item kufumo items:
        total = total + item
        biggest = bigger(biggest, item)
        muveze item % 2 == 1:
            odds = odds + 1
    pieces: vorito[galozi] = []
    zomagu index kufumo range(3):
        pieces.append(galozi(items[index]))
    label: galozi = ":".join(pieces)
    print("TOTAL " + galozi(total))
    print("BIGGEST " + galozi(biggest))
    print("ODDS " + galozi(odds))
    print("LABEL " + label)


main()
